src/histogram-date-range.ts

```diff
@@ -984,6 +984,27 @@ export class HistogramDateRange extends LitElement {
     `;
   }
 
+  private get histogramAccessibility(): TemplateResult {
+    let rangeText: string;
+    if (this.minSelectedDate && this.maxSelectedDate) {
+      rangeText = `from ${this.minSelectedDate} to ${this.maxSelectedDate}`;
+    } else if (this.minSelectedDate) {
+      rangeText = `from ${this.minSelectedDate}`;
+    } else if (this.maxSelectedDate) {
+      rangeText = `up to ${this.maxSelectedDate}`;
+    } else {
+      rangeText = 'for all available dates';
+    }
+
+    const titleText = `Filter results distribution ${rangeText}`;
+
+    const descText =
+      `This chart shows the distribution of search results ${rangeText}. The bars represent result counts for each time period within the selected date range.`.trim();
```

Comment:
I might also include the word "date" in that first sentence so they don't have to read to the end to get to "selected date range"